<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Lemon</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>TicTacToe </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gamelabel1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gamelabel2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gamelabel3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gamelabel6">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gamelabel4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gamelabel5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gamelabel9">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>400</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gamelabel8">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>400</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gamelabel7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>360</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Orbitron</family>
      <pointsize>36</pointsize>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>PLAY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playAgainButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>360</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Orbitron</family>
      <pointsize>28</pointsize>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>PLAY AGAIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="winnerX">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>500</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bungee Inline</family>
      <pointsize>52</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>X has WON!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="winnerO">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>500</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bungee Inline</family>
      <pointsize>52</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>O has WON!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="drawLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>500</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bungee Inline</family>
      <pointsize>52</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>DRAW</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Code</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
